class.jetpack-connection-banner.php

```diff
@@ -128,6 +127,11 @@ public static function enqueue_banner_scripts() {
 		);
 	}
 
+	/**
+	 * Enqueues JavaScript and CSS for new connect-in-place flow.
+	 *
+	 * @since 7.2
```

Comment:
Wow, was it really introduced in 7.2?

Reply:
Oh my, it's a typo. It should be 7.7, right @tyxla?

Reply:
It's now updated.

Reply:
👍 Right 👍